src/components/tables/RefreshTokens/Rows.tsx

```diff
@@ -0,0 +1,47 @@
+import { TableCell, TableRow, useTheme } from '@mui/material';
+import { RefreshTokenQuery } from 'api/tokens';
+import { getEntityTableRowSx } from 'context/Theme';
+import { FormattedMessage } from 'react-intl';
+import TimeStamp from '../cells/TimeStamp';
+import RevokeTokenButton from '../cells/refreshTokens/RevokeToken';
+
+interface RowsProps {
+    data: RefreshTokenQuery[];
+}
+
+interface RowProps {
+    row: RefreshTokenQuery;
+}
+
+function Row({ row }: RowProps) {
+    const theme = useTheme();
+
+    return (
+        <TableRow hover sx={getEntityTableRowSx(theme)}>
+            <TimeStamp time={row.created_at} enableRelative />
```

Comment:
What is driving this not following how we handle the `published` column on the entity tables?

![image](https://github.com/estuary/ui/assets/270078/1c9f7940-f8cd-48a6-8c0a-b01670220619)
`vs`
![image](https://github.com/estuary/ui/assets/270078/754ccb69-65e3-4f98-a9df-a5716b3bc5b2)

Reply:
The formatting is closest to what Dave and Johnny specified in their document without being novel.

<img width="537" alt="aa" src="https://github.com/estuary/ui/assets/77648584/13abd219-2139-4d1f-9215-24fd5b08275c">